Net31Solution/SensorApp/ConfigReader.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.IO;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.Json;
+using Microsoft.Extensions.Configuration.Xml;
+using System.Threading.Tasks;
+
+namespace SensorApp
+{
+    public static class ConfigReader
+    {
+        public static PressureSensorConfig PConfig { get; set; }
```

Comment:
There is no sense to have separate classes for sensor configs, all of them have the same structure. 
Use single SensorConfiguration class. Keep in mind that numerous sensor instances could be configured